Automattic/wp-calypso
# Pricing page rework | Remove Feature Flag and Clean up unused code

Since we have reached a point of stability for the Pricing Page Rework v1 project. It's the time to revisit and clean up the code and remove the feature flag and its usage.

#### Proposed Changes

* Remove feature flag usage from the code
* Remove feature flag from config JSON files
* Remove unused code from old pricing page

#### Testing Instructions

- Do any one of these
    * Click on Jetpack Cloud live link below and wait for the redirect to complete, then goto `/pricing`
    * or boot up this PR 
        * Run `git fetch && git checkout pricing-page-rework/clean-up`
        * Run `yarn start-jetpack-cloud`
        * Goto [http://jetpack.cloud.localhost:3000/pricing](http://jetpack.cloud.localhost:3000/pricing)
- Confirm that everything is the same as before
- Confirm the same for Calypso Blue - `/plans/:site`

#### Pre-merge Checklist





Completes 1202858161751496-as-1202835600809438/f

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202835600809438
